hc/api/tests/test_check_model.py

```diff
@@ -36,3 +43,12 @@ def test_paused_check_is_not_in_grace_period(self):
         self.assertFalse(check.in_grace_period())
 
     ### Test that when a new check is created, it is not in the grace period
+    def test_new_check_is_not_in_grace_period(self):
+        check = Check()
+
+        check.status = "up"
```

Comment:
A new check should have a status of "new" by default, no need to set it or the here last ping. I think the objective here is to assert that the new Check is not in the grace period by default.